google-cloud-bigquery/lib/google/cloud/bigquery/dataset.rb

```diff
@@ -668,6 +669,90 @@ def tables token: nil, max: nil
           Table::List.from_gapi gapi, service, dataset_id, max
         end
 
+        ##
+        # Retrieves an existing model by ID.
+        #
+        # @param [String] model_id The ID of a model.
+        # @param [Boolean] skip_lookup Optionally create just a local reference
+        #   object without verifying that the resource exists on the BigQuery
+        #   service. Calls made on this object will raise errors if the resource
+        #   does not exist. Default is `false`. Optional.
+        #
+        # @return [Google::Cloud::Bigquery::Model, nil] Returns `nil` if the
+        #   model does not exist.
+        #
+        # @example
+        #   require "google/cloud/bigquery"
+        #
+        #   bigquery = Google::Cloud::Bigquery.new
+        #   dataset = bigquery.dataset "my_dataset"
+        #
+        #   model = dataset.model "my_model"
+        #   puts model.name
```

Comment:
I think `model.model_id` is better for these examples, since it is required to be provided to `CREATE MODEL`.

Reply:
What do you think about `model_type`?

Reply:
Changed to `model.model_id`.